### ISSUE TITLE
[C++] Variadic template parameter pack expansion breaks function definition parser

### ISSUE THREAD

**Issue by merlinthered** [sublimehq/Packages#693]
### Working:
```cpp
template<typename T> void SomeClass<T>::function();
```
here, `function` is correctly parsed as a function name.
#### scopes:
```
source.c++ 
meta.namespace.c++ 
meta.block.c++ 
meta.function.c++ 
entity.name.function.c++
```

### Not working:
```cpp
template<typename T...> void SomeClass<T...>::function();
```
here, `function` is not parsed as a function name.
#### scopes:
```
source.c++ 
meta.namespace.c++ 
meta.block.c++ 
meta.function-call.c++ 
variable.function.c++
```

**Comment by wbond** [comment]
Thanks for this bug report!

**Comment by wbond** [comment]
This is fixed by 6fa9698384e57e68e70d3f5d03d7331eaa9edd8d and will be part of the next dev build
